.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Documentation:
+  Enabled: false
```

Comment:
Recordá agregar newlines al final de todos los archivos.
(Es raro que rubocop no haya avisado "no newline at end of file")